### ISSUE TITLE
Lighthouse Accessibility failure

### ISSUE THREAD

**Issue by JustinHermanM** [magento/magento2#39054]
### Preconditions and environment

- 2.4.7-p1 
- Chrome inspect , click on light house , 
Nav sections fail accessibility checker

### Steps to reproduce

Using Chrome with Lighthouse extension installed. 
Inspect and select Lighthouse 
Mode = Navigation (default)
Device = Desktop 
Categories = Accessibility 
Analyze page load 

### Expected result

Pass with Accessibility score of 100 

### Actual result

Accessibility score of 95 
Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.
div.section-items.nav-sections-items.mage-tabs-disabled
div#store.menu.section-item-content.nav-sections-item-content

### Additional information

The parent - 
<div class="section-items nav-sections-items mage-tabs-disabled" role="tablist">
one of the children 
     <div class="section-item-content nav-sections-item-content" id="store.menu" data-role="content" role="tabpanel" aria-hidden="false">
</div>
</div>

I believe the child should have the role="tab" to pass Accessibility

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by engcom-Bravo** [comment]
Hi @JustinHermanM,

Thanks for your reporting and collaboration.

We have verified the issue in Latest 2.4-develop instance and the issue is reproducible.Kindly refer the screenshots.

**Steps to reproduce**

- Using Chrome with Lighthouse extension installed.
- Inspect and select Lighthouse
- Mode = Navigation (default)
- Device = Desktop
- Categories = Accessibility
- Analyze page load

<img width="1434" alt="Screenshot 2024-08-20 at 12 01 49" src="https://github.com/user-attachments/assets/946156db-d22a-40c3-9535-7a7c3c44da89">



Hence Confirming the issue.

Thanks.

**Comment by hostep** [comment]
@engcom-Bravo: can I ask you to increase the priority here? Because here in Europe [a law will start applying from June 2025 onwards](https://ec.europa.eu/social/main.jsp?catId=1202&intPageId=5581&langId=en) to force websites to adhere to accessibility rules.
And since Magento has only one big release scheduled before that date (version 2.4.8, in April 2025), it would be good if we can get this fixed in 2.4.8 if possible.

**Comment by Serfe-com** [comment]
@hostep you can review the fix for this issue on [!39164](https://github.com/magento/magento2/pull/39164) where we got this into 100% in accessibility on Lighthouse. You can use that MR as a patch until Magento team merges it into the core.
